cartridges/app_storefront_training/cartridge/static/default/js/cart.js

```diff
@@ -0,0 +1,150 @@
+/******/ (function(modules) { // webpackBootstrap
```

Comment:
Don't commit .js files inside of the static folder.
When the branch reaches the development environment its own build is triggered